core/base/document.php

```diff
@@ -731,6 +731,7 @@ protected function get_initial_config() {
 			if ( Plugin::$instance->experiments->is_feature_active( 'container' ) ) {
 				$container_config = [
 					'container' => Plugin::$instance->elements_manager->get_element_types( 'container' )->get_config(),
+					'a-container' => Plugin::$instance->elements_manager->get_element_types( 'a-container' )->get_config(),
```

Comment:
Check the Experiment before adding it here

Reply:
The Atomic widgets experiment.